packages/syntax/tests/did.test.ts

```diff
@@ -64,4 +66,34 @@ describe('DID permissive validation', () => {
     expectValid('did:key:zQ3shZc2QzApp2oymGvQbzP8eKheVshBHbU4ZYjeXqwSKEn6N')
     expectValid('did:ethr:0xb9c5714089478a327f09197987f16f9e5d936e8a')
   })
+
+  it('conforms to interop valid DIDs', () => {
+    const lineReader = readline.createInterface({
+      input: fs.createReadStream(
+        `${__dirname}/interop-files/did_syntax_valid.txt`,
+      ),
+      terminal: false,
+    })
+    lineReader.on('line', (line) => {
+      if (line.startsWith('#') || line.length == 0) {
+        return
+      }
```

Comment:
I've had good experiences with TOML, which has good commenting. An advantage of any wrapped format is that any weird bytes are obvious because of quote encoding.

The downside is that we want whatever we used to be as easy to parse and use in any implementation language, which is why I stuck to text and JSON so far. I think having it be the exact same file on disk across projects is a big advantage. But maybe TOML/JSON5 isn't a heavy lift in most languages these days.

Reply:
Oh yeah TOML could work too. Good call that if you want them portable we need to consider the universality of the format 👍